addons/binding/org.openhab.binding.exec/META-INF/MANIFEST.MF

```diff
@@ -27,3 +29,4 @@ Import-Package:
  org.osgi.framework,
  org.slf4j
 Service-Component: OSGI-INF/*.xml
+Require-Bundle: org.eclipse.smarthome.core

```

Comment:
The manifest must not contain any `Require-Bundle` entries. Instead, `Import-Package` must be used. [[B.4](http://docs.openhab.org/developers/development/guidelines.html#b-osgi-bundles)]

This means you also have to reference specific packages in the `Import-Package` instead of the top-level packages needed for `Require-Bundle`